simulator-samples/sample-rest/src/test/java/com/consol/citrus/simulator/SimulatorRestIT.java

```diff
@@ -134,7 +135,8 @@ public void testDefaultRequest() {
     /**
      * Sends some intervening request to server expecting positive response message.
      */
-    @CitrusTest
+    //@CitrusTest
+    @Ignore("Fails with citrus 3.0.0")
```

Comment:
we should take a look at this.

Reply:
[Fixed](https://github.com/citrusframework/citrus-simulator/pull/102/commits/52359df58b7f729514365f88d88c50a484d935e7)